addons/aiCfgFixes/CfgAmmo.hpp

```diff
@@ -0,0 +1,613 @@
+class CfgAmmo {
+    class Default;
+    class ShellCore: Default {
+        audibleFire = 16;
+    };
+    class BulletCore: Default {
+        audibleFire = 16;
+    };
+    class SubmunitionCore: Default {
+        audibleFire = 16;
+    };
+    class RocketCore: Default {
+        audibleFire = 32;
+    };
+    class MissileCore: Default {
+        audibleFire = 32;
+    };
+    class Grenade: Default {
+        audibleFire = 0.25;
+    };
+    class GrenadeCore: Default {
+        audibleFire = 0.25;
+    };
+    class LaserCore: Default {
+        audibleFire = 32;
+    };
+    class TimeBombCore: Default {
+        audibleFire = 0;
+    };
+    class ShotDeployCore: Default {
+        audibleFire = 16;
+    };
+    class ShotgunCore: Default {
+        audibleFire = 16;
+    };
+    class ShellBase: ShellCore {
+        audibleFire = AI_AUDIBLE_FIRE_0;
+        GVAR(macroUsed) = "AI_AUDIBLE_FIRE_0";
```

Comment:
Is there any non debug reason to have this value?

Reply:
The Macro -> So if we find that the value I used is too extreme we can tweak all applicable values en-masse

`GVAR(macroUsed)` -> needed for the script to add the appropriate macro to the `avaliableFire` variable